text/0043-phrase-query.md

```diff
@@ -0,0 +1,172 @@
+            "author": "Martin Denton"
+        },
+        {
+            "title": "Plays and Playwrights 2008",
+            "author": "Robert Attweiler"
+        },
+        ...
+        {
+            "title": "The Best American Short Plays 2006-2007",
+            "author": "Barbara Parisi"
+        },
+        {
+            "title": "New Playwrights: The Best Plays of 2000",
+            "author": "D. L. Lepidus"
+        },
+        {
+            "title": "Women Heroes: Six Short Plays from the Women's Project",
+            "author": "Julia Miles"
+        }
+    ],
+    "nbHits": 31,
+    "exhaustiveNbHits": false,
+    "query": "Plays and Playwrights 2002",
+    "limit": 20,
+    "offset": 0,
+    "processingTimeMs": 7
+}
+```
+
+To use the Phrase Query syntax, simply surround the contiguous search terms with the characters `"`.
+
+Using the Phrase Query syntax this way, with `q` equals to `"Plays and Playwrights 2002"`, will lead to have only one result because the title is exactly written like that.
+
+> Note that it's case insensitive. So, if i search with `"plays and playwrights 2002"`, this will lead to the same result.
+
+The value between the `"` operators will be searched without:
+
+- typography
+- n-gram
+- wordsplit
+- prefix
+- synonym
+- proximity
+
+
+```
+{
+    "hits": [
+        {
+            "title": "Plays and Playwrights 2002",
+            "author": "Martin Denton"
+        }
+    ],
+    "nbHits": 1,
+    "exhaustiveNbHits": false,
+    "query": "\"Plays and Playwrights 2002\"",
+    "limit": 20,
+    "offset": 0,
+    "processingTimeMs": 0
+}
+```
+
+So now let's say that i want to search title strictly mentioning `"African American"` but speaking about poetry. The Phrase query syntax can be used in conjunction with the basic syntax.
+
+The query can be exprimed like that `"African American" poem`
+
+```
+{
+    "hits": [
+        {
+            "title": "100 Best African American Poems with CD",
+            "author": "Nikki Giovanni"
+        },
+        {
+            "title": "The African American Experience: Black History and Culture Through Speeches, Letters, Editorials, Poems, Songs, and Stories",
+            "author": "Kai Wright"
+        },
+        {
+            "title": "African American Literature (Penguin Academics Series)",
+            "author": "Keith Gilyard"
+        },
+        {
+            "title": "African-American Poetry: An Anthology, 1773-1930",
+            "author": "Joan R. Sherman"
+        },
+        {
+            "title": "African-American Literature: A Brief Introduction and Anthology",
+            "author": "Al Young"
+        },
+        {
+            "title": "Early African American Classics (Barnes &amp; Noble Classics Series)",
+            "author": "Barnes &amp; Noble"
+        },
+```
+
+As you can see in the results, the presence or absence of one or more soft separators such as `-` between two words does not affect the query match of the phrase and the document.
```

Comment:
What are the other "soft separators"?

Reply:
I've updated the spec with the complete list of soft separators 🤓